event-adapter/cmd/provider/translator.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provider
+
+import (
+	"encoding/json"
+	"fmt"
+	sd "google.golang.org/api/logging/v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	api "k8s.io/client-go/pkg/api/v1"
+)
+
+type eventTime struct {
+	Timestamp metav1.Time `json:"timestamp,omitempty"`
+}
+
+type eventsTranslator struct {
+	eventsList    *api.EventList
```

Comment:
What's the benefit of having eventsList here in comparison to having only a local variable in function "translateToEventList"? The same applies to the other two fields. Going further into that, do you even need eventsTranslator structure? Can you remove it?

Reply:
Removed